### ISSUE TITLE
[XamarinProfiler-Windows]: Accessibility: Color Contrast: MAS1.4.3: "Symbol Name" coulmn text in Call Tree Table of Call Tree tab is not having color contrast ratio of 4.5:1 in both "Allocations/Time Profiler" profiling pages

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#430]
### Pre-Requisite:
1. Install Xamarin Profiler.
2. Launch Profiler through ".mlpd" file.
### Steps to Reproduce

Scenario 1:
1. Follow the Pre-requisite to launch the xamarin profiler.
2. Select the available instruments to "Allocations" and hit "Start Profiling"button.
3. This will invoke "Allocations" profiling page. Navigate to "Call Tree" tab and invoke it.
4. Verify the color contrast ratio of "Text" under "Symbol Name" column of Call tree.
Scenario 2:
1. Follow the Pre-requisite to launch the xamarin profiler.
2. Select the available instruments to "Time Profiler" and hit "Start Profiling"button.
3. This will invoke "Time Profiler" profiling page. 
4. Verify the color contrast ratio of "Text" under "Symbol Name" column of Call tree.
Note: Same issue is observed in "Samples" tab. Please refer the below steps.
5. Navigate to "Samples" tab and invoke it. This will invoke "Samples" tab of Time Profiler.
6. Verify the color contrast ratio of "Name" column text of "All Samples" table.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1
### Actual Behavior
Color contrast ratio is less than 4.5:1 i.e, 2.7:1 - 4.2:1
See the screenshots attached
### User Impact
Low vision user will feel difficult in seeing the text
### Screenshots
![4 allocations - call tree](https://user-images.githubusercontent.com/22956130/41711306-74cc6e5e-7565-11e8-80c3-62d802807436.png)
![4 time profiler - call tree](https://user-images.githubusercontent.com/22956130/41711330-83bb5b28-7565-11e8-9b01-8c4ee3b5d14d.png)
![4 time profiler - samples](https://user-images.githubusercontent.com/22956130/41711342-89099a68-7565-11e8-9984-a7aea142550b.png)
![4 time profiler - sample 2](https://user-images.githubusercontent.com/22956130/41711346-8e6475fa-7565-11e8-8214-33c5064c8071.png)

### Environment
Microsoft Windows version: 1803(OS Build 17134.112)
Xamarin Profiler version: 1.6.3.342

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Profiler-Windows
Wipro
